app/Models/Competition.php

```diff
@@ -31,16 +31,19 @@ public function users()
     /**
      * Gets a CSV export of all the users in this competition.
      *
+     * @param \Gladiator\Repositories\UserRepositoryContract $repository
      * @return \League\Csv\ $csv
      */
-    public function getCSVExport()
+    public function getCSVExport($repository)
```

Comment:
We have a few services in repo already. Check the Registrar as an example that doesn't to be formally defined as a Laravel Service Provider, since it'll likely be a simple class with lots of methods.